WebRoot/css/zm.css

```diff
@@ -6585,3 +6579,19 @@ a.ZmLinkDisabled {
 	@MailAttachmentsAllText@
 }
 
+/* Create Contact Group dialog */
+#CreateContactGroupDialog {
+	@CreateContactGroupDialog@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet {
+	@CreateContactGroupDialogTable@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet td {
+	@CreateContactGroupDialogCell@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet td input[type='text'] {
```

Comment:
Sure, we can discuss it. But if you see CSS rules above, we get some understanding of what elements those rules will target to. For instance, table, all inputs inside table etc.  `#CreateContactGroupDialog_name ` looks bit odd here and we may not get clear idea at first sight what that selector's will refers to.

I understands it is based on one's opinion and everyone may has different opinion on this. Should you feel I should change selector, let me know. I will make suggested changes.

Thanks